umm-spec-lib/test/cmr/umm_spec/test/date_util_test.clj

```diff
@@ -3,6 +3,7 @@
   (:require
    [clj-time.format :as f]
    [clojure.test :refer :all]
+   [cmr.common.time-keeper :as time-keeper]
```

Comment:
I don't remember why, but @chris-durbin or @ygliuvt told me to use `[cmr.system-int-test.utils.dev-system-util :as dev-system-util]` here. Something to do with the dev-system time keeper properly resetting time after you freeze.

It works effectively the same: `(dev-system-util/freeze-time! "2010-01-01T10:00:00Z")`